charmhelpers/contrib/openstack/context.py

```diff
@@ -1034,6 +1034,9 @@ def __call__(self):
         if config('haproxy-connect-timeout'):
             ctxt['haproxy_connect_timeout'] = config('haproxy-connect-timeout')
 
+        if config('haproxy-enable-proxyv2'):
+            ctxt['haproxy_enable_proxyv2'] = config('haproxy-enable-proxyv2')
```

Comment:
this can be set to True since the config is actually boolean. Just makes it clearer that it doesn't need to be set to a more specific value.